app/Main.js

```diff
@@ -25,7 +25,10 @@ try {
 function runApp(initial_state) {
     console.log('Initial state', initial_state);
     const config = initial_state.offchain.config
-    steem.config.set('websocket', config.ws_connection_client);
+    if(process.env.USE_JSONRPC)
```

Comment:
yes but the client won't have process.env - maybe that just means this always falls through in the SPA?